Branch-SDK/src/main/java/io/branch/referral/BranchJsonConfig.java

```diff
@@ -202,4 +204,18 @@ public String getAPIUrl() {
             return null;
         }
     }
+
+    @Nullable
+    public String getFbAppId() {
+        if (mConfiguration == null) return null;
+
+        try {
+            if (!mConfiguration.has(BranchJsonKey.fbAppId.toString())) return null;
```

Comment:
Nit: every statement within a function/control block should be enclosed with {} and on a new line